src/main/java/org/zalando/nakadi/controller/SchemaController.java

```diff
@@ -0,0 +1,37 @@
+package org.zalando.nakadi.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.zalando.nakadi.service.Result;
+import org.zalando.nakadi.service.SchemaService;
+import org.zalando.problem.spring.web.advice.Responses;
+
+@RestController
+public class SchemaController {
```

Comment:
I have SchemaControllerAT test. Would it be enough for you if I test problem cases(bad request) inside that test?